### PR TITLE
Add ParseStethoInterceptor

### PR DESCRIPTION
**Pull request by wangmengyan95**
Currently rely on manually test, using three http clients and check normal and GZIP response.

### CODE REVIEW

File: Parse/src/main/java/com/parse/ParseStethoInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+            .setHeaders(response.getAllHeaders())
+            .setReasonPhase(response.getReasonPhrase())
+            .setStatusCode(response.getStatusCode())
+            .setContent(responseStream)
+            .build();
+      }
+    }
+    return response;
+  }
+}
```
Comment (grantland) [inline Parse/src/main/java/com/parse/ParseStethoInterceptor.java]: nit: newline at EOF

**Comment by grantland** [comment]
LGTM, let's get Josh looking at this.

In addition, do you think we should put this and `LogcatInterceptor` in a separate module? That way developers don't have to ship this extra code, but can add another Maven dependency when they want to use it?

**Comment by wangmengyan95** [comment]
Discuss offline, will split `ParseLogInterceptor` and `ParseStethoInterceptor` to a separate module.

File: Parse/src/main/java/com/parse/ParseStethoInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,278 @@
+package com.parse;
```
Comment (grantland) [inline Parse/src/main/java/com/parse/ParseStethoInterceptor.java]: Add license headers

**Pull request merged**